### PR TITLE
wavy:0.1.0

### PR DESCRIPTION
**Pull request by Enter-tainer**
I am submitting
- [x] a new package
- [ ] an update for a package

Description: This package use jogs to run wavedrom in typst. [wavedrom](https://wavedrom.com/) is a popular js library to draw waveforms. With this people can use wavedrom inside typst.

I have read and followed the submission guidelines and, in particular, I
- [x] selected a name in conformance with the guidelines
- [x] added a `typst.toml` file with all required keys
- [x] added a `README.md` with documentation for my package
- [x] have chosen a license and added a `LICENSE` file or linked one in my `README.md`
- [x] ensured that my submission does not infringe upon the rights of a third party
- [x] tested my package locally on my system and it worked
- [x] named this PR as `name:version` of the submitted package
- [x] agree that my package will not be removed without good reason

### CODE REVIEW

File: packages/preview/wavy/0.1.0/typst.toml [hunk 1]
```diff
@@ -0,0 +1,17 @@
+[package]
+name = "wavy"
+version = "0.1.0"
+entrypoint = "lib.typ"
+authors = ["Wenzhuo Liu"]
+license = "MIT"
+description = "Draw digital timing diagram in Typst using Wavedrom."
+
```
Comment (laurmaedje) [inline packages/preview/wavy/0.1.0/typst.toml:8]: could you exclude `wavg.svg`? The README itself should be included, but the example SVG isn't necessary (it's also not that small actually at 112KB).

**Comment by laurmaedje** [comment]
Would it makes sense to minify the JS file since it's basically an executable file anyway?

**Comment by Enter-tainer** [comment]
The js is minified and its size reduced to 220K now

**Comment by laurmaedje** [comment]
Great, thanks!

**Comment by laurmaedje** [comment]
I didn't think of it earlier, but I believe we are lacking attribution for `wavy.js`.

**Comment by Enter-tainer** [comment]
> I didn't think of it earlier, but I believe we are lacking attribution for `wavy.js`.

do you mean the license thing? i can take a look

**Comment by laurmaedje** [comment]
What I typically do is create a NOTICE file, it's a bit more lightweight and should fulfill the same purpose. typst also has one.

**Comment by laurmaedje** [comment]
Since in this case it's both MIT maybe it would even suffice to add their copyright statement to the license here. I guess the implications on the scope of the license would be a bit different though.

**Comment by Enter-tainer** [comment]
> What I typically do is create a NOTICE file, it's a bit more lightweight and should fulfill the same purpose. typst also has one.

ok i will do it. btw this may apply to other plugin packages. svg-bob graphviz layout-rs... you may update the package pr template to emph this.

**Comment by laurmaedje** [comment]
yeah, it's true. I need to read up on attribution for binaries like wasm, but you're right that it should be the same...

**Pull request merged**